### DISCUSSION TITLE
A few paper cuts in links

### DISCUSSION THREAD

**Discussion by henryiii** [scikit-hep/pyhf#1213] [Ideas]
I've noticed a few oddities in clicking around the repo, and wanted to point them out here. Very minor, and mostly preference, but maybe one or more would be helpful?

**Comment by henryiii** [comment]
The conda forge badge links to the anaconda download page, rather than the feedstock on conda-forge. The feedstock is more informative and useful to me, and it also links to the download page too, so it seems to be a better place to link.

**Reply by matthewfeickert** [reply]
That's a good point. I can see how [![Conda-forge version](https://img.shields.io/conda/vn/conda-forge/pyhf.svg)](https://github.com/conda-forge/pyhf-feedstock) provides more helpful info.

**Comment by henryiii** [comment]
The "Fork me on GitHub" link _actually_ starts a fork - 99% of the time I click on that it's just to get to the repo as fast as possible - "edit me" and "fork me" links are _always_ at the top right, so it's the one reliable way to get to GitHub quickly. Actually starting the fork process is not necessary, most repos don't have that banner doesn't start the fork process. Just like the star me buttons also don't actually add the star, and the edit me buttons don't usually start the edit, they just take you to the step before those actions. (not always).

**Reply by henryiii** [reply]
Note that your edit me button does work the way I'd expect, just taking you to the page but not starting the edit process.

**Reply by matthewfeickert** [reply]
>  99% of the time I click on that it's just to get to the repo as fast as possible - "edit me" and "fork me" links are _always_ at the top right, so it's the one reliable way to get to GitHub quickly.

Yeah this is also my thoughts. I just use it so rarely that I haven't gotten around to changing it, but this is a good reminder. Thanks!
